vite.config.js

```
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  build: {
+    // 本番ビルドではソースマップを無効化し、ファイルサイズを削減します。
+    sourcemap: false,
+    // コード圧縮にTerserを使用し、オプションを設定します。
+    minify: 'terser',
```

ESLint

[error] Replace `····` with `········`

(prettier/prettier)